### PR TITLE
Merge in upstream changes

### PR DESCRIPTION
**Pull request by adrianmoisey**
This merges in all the upstream changes of Jenkins into our repo. 

I'd like to get this merged and uploaded to the chef server so that I can test it out on an int-env, trying it out with chef-solo was too painful.

The idea is to migrate production to use this cookbook over our hacky thing.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,100 @@ jenkins Cookbook CHANGELOG
 ==========================
 This file is used to list changes made in each version of the jenkins cookbook.
 
+v2.2.2 (2015-01-15)
+-------------------
+### Bug
+- Gem::Version raising ArgumentError for weirdly versioned Jenkins plugins
+- Force UTF-8 encoding when parsing update center JSON
+- README grammer fixes
```
Comment (jouberthenk) [inline CHANGELOG.md]: grammer

Comment (adrianmoisey) [inline CHANGELOG.md]: :) https://github.com/opscode-cookbooks/jenkins/pull/346

**Comment by jouberthenk** [comment]
Doesn't look like anything dangerous (no weird dependencies or anything). Maybe leave until monday so peeps across the pond can have a look.

:+1:

**Comment by adrianmoisey** [comment]
> Maybe leave until monday so peeps across the pond can have a look.

Since we're not promoting the version in the environment I'd like to merge this now, so that I can start testing on an int-env.

**Comment by jouberthenk** [comment]
I did say `maybe` so go ahead :rocket:

**Comment by adrianmoisey** [comment]
pew pew!

**Pull request merged**